doc/bosh-release-development.md

```diff
@@ -0,0 +1,162 @@
+       - [Uploading The Image](#uploading-the-image)
+       - [Modify Kubecf to Use the New Image](#modify-kubecf-to-use-the-new-image)
+  -  [Integrating the Release in Kubecf](#integrating-the-release-in-kubecf)
+       - [BPM](#bpm)
+       - [Operation Files](#operation-files)
+  -  [Testing With Kubecf](#testing-with-kubecf)
+
+## Preparing the Release Image
+
+BOSH release authors, who want to test their development code with the
+Quarks operator, need to build a Docker image from their release.
+This can be done with `fissile`.  Afterwards upload the image to a
+cluster for testing it, e.g. with Kubecf.
+
+### Building a Docker Image with Fissile
+
+Build the BOSH release first and convert it with [fissile].
+
+To generate a docker image from the BOSH release, you should use the
+following subcommand:
+
+```sh
+fissile build release-image
+```
+
+For more information on how to use the command, please refer to the
+related [documentation]. For a real example, see [build.sh].
+
+[fissile]:       https://github.com/cloudfoundry-incubator/fissile
+[documentation]: https://github.com/cloudfoundry-incubator/fissile/blob/develop/docs/build-docker-imgs.md
+[build.sh]:      https://github.com/cloudfoundry-incubator/cf-operator-ci/blob/e83e46548787ee740ea1918182604faaa5cddf8f/pipelines/release-images/tasks/build.sh#L34
+
+### Uploading The Image
+
+Depending on your cluster, you will need a way to get the locally
+built image into the Kubernetes registry.
+
+With __minikube__ you can build directly on minikube's Docker. Switch
+to that docker daemon by running `eval $(minikube docker-env)`, before
+you build the image with `fissile`.
+
+With __kind__, you need to use `kind load docker-image` after building
+the image, to make it available, i.e.:
+
+```sh
+kind load docker-image docker.io/org/nats:0.1-dev
+```
+
+### Modify Kubecf to Use the New Image
+
+Add an operations file to Kubernetes with the new image location. The
+example below uses NATS as the example for a BOSH release.
+
+```yaml
+kubectl apply -f - <<EOF
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nats-dev
+data:
+  ops: |
+- type: replace
+  path: /releases/name=nats?
+  value:
+    name: nats
+    url: docker.io/org/nats
+    version: 0.1-dev
+    sha1: ~
+EOF
+```
+
+Then, when running `helm install scf` refer to that image:
+
+```sh
+helm install ... --set 'operations.custom={nats-dev}'
+```
+
+__Note__: You can also unpack the helm release and modify it directly.
+There is no need to zip the release again, as `helm install scf/` is
+able to install the unpacked release.
+
+Note further that the above is an example of how to use the first kind
+of customization feature noted in the main [README](Contribute.md#customization).
+
+## Integrating the Release in Kubecf
+
+With Quarks and Kubecf BOSH releases can largely be used just the same
+as with a BOSH director. There are a few things Quarks offers however
+to make adaption to the kubernetes environment easier.
```

Comment:
The last sentence has a few issues, here is the correct one:
```
There are a few things Quarks offers, however, to make the adaptation to the Kubernetes environment easier.
```

Reply:
Fixed with commit 9918b20dcb257d8865a289cee9653b710a4662a0.